src/index.html

```diff
@@ -5,8 +5,413 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img src="images/logo.png" alt="moyo logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a href="index.html" class="nav__link is-active">apple</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="index.html" class="nav__link">samsung</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="index.html" class="nav__link">smartphones</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="index.html"
+            class="nav__link"
+            data-qa="nav-hover">laptops & computers</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="index.html" class="nav__link">gadgets</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="index.html" class="nav__link">tablets</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="index.html" class="nav__link">photo</a>
+          </li>
+
+          <li class="nav__list-item">
+            <a href="index.html" class="nav__link">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog">
+        <div class="product" data-qa="card">
+          <img
+              class="product__photo"
+              src="./images/imac.jpeg"
+              alt="APPLE iMac photo"
+            >
+          <h2 class="product__title-text">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product__code-text">
+            Product code: 195434
+          </p>
+
+          <div class="product__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="product__reviews-text">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="product__price">
+            <p class="product__price-text">
+              Price:
+            </p>
+
+            <p class="product__price-value">
+              $2,199
+            </p>
+          </div>
+
+          <a
+            class="product__button"
+            href="index.html"
+            data-qa="card-hover"
+            >
+            BUY
+          </a>
+
+        </div>
+        <div class="product">
```

Comment:
Add a blank line between the cards
```suggestion
        </div>
        
        <div class="product">
```